versions.yml

```diff
@@ -188,21 +209,52 @@ modGroups:
       desc: Adds Texture Features to entities in Optifine compatible format.
       url: https://cdn.modrinth.com/data/BVzZfTc1/versions/1Y3lxOX6/entity_texture_features_fabric_1.20.2-4.5.1.jar
 
+    - name: Continuity
+      license: LGPL-3
+      homepage: https://modrinth.com/mod/continuity
+      desc: Provides default connected textures
+      url: https://cdn.modrinth.com/data/1IjD5062/versions/JXhQlDZl/continuity-3.0.0-beta.4%2B1.20.2.jar
+      requires: [Indium]
```

Comment:
I don't see Indium in the dependencies list

Reply:
Indium is required when using Sodium, that's why it is not listed as Continuity does not require Sodium. Without Indium it is as if it is not installed.

Reply:
I would still rather have indium installed in cases where it causes issues. And I don't see indium anywhere in the list

Reply:
Indium is in core mods, meaning it is always installed, there is no option to disable it

Reply:
Didn't see that. Okay, this should work then